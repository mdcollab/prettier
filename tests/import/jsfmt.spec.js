run_spec(__dirname, null, ["typescript"]);
run_spec(__dirname, { bracketSpacing: false }, ["typescript"]);
