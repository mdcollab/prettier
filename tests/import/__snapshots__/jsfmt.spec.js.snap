// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`brackets.js 1`] = `
"import {
  runTaskForChanged,
  description,
  someOtherLabel,
  thatMakes,
  itGo,
  multiLine,
  andMore,
  soWeCanGetItTo80Columns
} from '.';
import {fitsIn, oneLine} from '.';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import {
  runTaskForChanged,
  description,
  someOtherLabel,
  thatMakes,
  itGo,
  multiLine,
  andMore,
  soWeCanGetItTo80Columns
} from \\".\\";
import { fitsIn, oneLine } from \\".\\";
"
`;

exports[`brackets.js 2`] = `
"import {
  runTaskForChanged,
  description,
  someOtherLabel,
  thatMakes,
  itGo,
  multiLine,
  andMore,
  soWeCanGetItTo80Columns
} from '.';
import {fitsIn, oneLine} from '.';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import {
  runTaskForChanged,
  description,
  someOtherLabel,
  thatMakes,
  itGo,
  multiLine,
  andMore,
  soWeCanGetItTo80Columns
} from \\".\\";
import {fitsIn, oneLine} from \\".\\";
"
`;

exports[`brackets.js 3`] = `
"import {
  runTaskForChanged,
  description,
  someOtherLabel,
  thatMakes,
  itGo,
  multiLine,
  andMore,
  soWeCanGetItTo80Columns
} from '.';
import {fitsIn, oneLine} from '.';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import {
  runTaskForChanged,
  description,
  someOtherLabel,
  thatMakes,
  itGo,
  multiLine,
  andMore,
  soWeCanGetItTo80Columns
} from \\".\\";
import { fitsIn, oneLine } from \\".\\";
"
`;

exports[`brackets.js 4`] = `
"import {
  runTaskForChanged,
  description,
  someOtherLabel,
  thatMakes,
  itGo,
  multiLine,
  andMore,
  soWeCanGetItTo80Columns
} from '.';
import {fitsIn, oneLine} from '.';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import {
  runTaskForChanged,
  description,
  someOtherLabel,
  thatMakes,
  itGo,
  multiLine,
  andMore,
  soWeCanGetItTo80Columns
} from \\".\\";
import {fitsIn, oneLine} from \\".\\";
"
`;

exports[`comments.js 1`] = `
"import { a //comment1
//comment2
//comment3
as b} from \\"\\";

import {
  a as //comment1
  //comment2
  //comment3
  b
} from \\"\\";

import {
  a as //comment2 //comment1
  //comment3
  b
} from \\"\\";

import {
  a as //comment3 //comment2 //comment1
  b
} from \\"\\";

import {
  // comment 1
  FN1, // comment 2
  /* comment 3 */ FN2,
  // FN3,
  FN4 /* comment 4 */
  // FN4,
  // FN5
} from \\"./module\\";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import {
  //comment1
  //comment2
  //comment3
  a as b
} from \\"\\";

import {
  //comment1
  //comment2
  //comment3
  a as b
} from \\"\\";

import {
  //comment2 //comment1
  //comment3
  a as b
} from \\"\\";

import {
  //comment3 //comment2 //comment1
  a as b
} from \\"\\";

import {
  // comment 1
  FN1, // comment 2
  /* comment 3 */ FN2,
  // FN3,
  FN4 /* comment 4 */
  // FN4,
  // FN5
} from \\"./module\\";
"
`;

exports[`comments.js 2`] = `
"import { a //comment1
//comment2
//comment3
as b} from \\"\\";

import {
  a as //comment1
  //comment2
  //comment3
  b
} from \\"\\";

import {
  a as //comment2 //comment1
  //comment3
  b
} from \\"\\";

import {
  a as //comment3 //comment2 //comment1
  b
} from \\"\\";

import {
  // comment 1
  FN1, // comment 2
  /* comment 3 */ FN2,
  // FN3,
  FN4 /* comment 4 */
  // FN4,
  // FN5
} from \\"./module\\";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import {
  //comment1
  //comment2
  //comment3
  a as b
} from \\"\\";

import {
  //comment1
  //comment2
  //comment3
  a as b
} from \\"\\";

import {
  //comment2 //comment1
  //comment3
  a as b
} from \\"\\";

import {
  //comment3 //comment2 //comment1
  a as b
} from \\"\\";

import {
  // comment 1
  FN1, // comment 2
  /* comment 3 */ FN2,
  // FN3,
  FN4 /* comment 4 */
  // FN4,
  // FN5
} from \\"./module\\";
"
`;

exports[`comments.js 3`] = `
"import { a //comment1
//comment2
//comment3
as b} from \\"\\";

import {
  a as //comment1
  //comment2
  //comment3
  b
} from \\"\\";

import {
  a as //comment2 //comment1
  //comment3
  b
} from \\"\\";

import {
  a as //comment3 //comment2 //comment1
  b
} from \\"\\";

import {
  // comment 1
  FN1, // comment 2
  /* comment 3 */ FN2,
  // FN3,
  FN4 /* comment 4 */
  // FN4,
  // FN5
} from \\"./module\\";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import {
  //comment1
  //comment2
  //comment3
  a as b
} from \\"\\";

import {
  //comment1
  //comment2
  //comment3
  a as b
} from \\"\\";

import {
  //comment2 //comment1
  //comment3
  a as b
} from \\"\\";

import {
  //comment3 //comment2 //comment1
  a as b
} from \\"\\";

import {
  // comment 1
  FN1, // comment 2
  /* comment 3 */ FN2,
  // FN3,
  FN4 /* comment 4 */
  // FN4,
  // FN5
} from \\"./module\\";
"
`;

exports[`comments.js 4`] = `
"import { a //comment1
//comment2
//comment3
as b} from \\"\\";

import {
  a as //comment1
  //comment2
  //comment3
  b
} from \\"\\";

import {
  a as //comment2 //comment1
  //comment3
  b
} from \\"\\";

import {
  a as //comment3 //comment2 //comment1
  b
} from \\"\\";

import {
  // comment 1
  FN1, // comment 2
  /* comment 3 */ FN2,
  // FN3,
  FN4 /* comment 4 */
  // FN4,
  // FN5
} from \\"./module\\";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import {
  //comment1
  //comment2
  //comment3
  a as b
} from \\"\\";

import {
  //comment1
  //comment2
  //comment3
  a as b
} from \\"\\";

import {
  //comment2 //comment1
  //comment3
  a as b
} from \\"\\";

import {
  //comment3 //comment2 //comment1
  a as b
} from \\"\\";

import {
  // comment 1
  FN1, // comment 2
  /* comment 3 */ FN2,
  // FN3,
  FN4 /* comment 4 */
  // FN4,
  // FN5
} from \\"./module\\";
"
`;

exports[`long-line.js 1`] = `
"import someCoolUtilWithARatherLongName from '../../../../utils/someCoolUtilWithARatherLongName';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import someCoolUtilWithARatherLongName
  from \\"../../../../utils/someCoolUtilWithARatherLongName\\";
"
`;

exports[`long-line.js 2`] = `
"import someCoolUtilWithARatherLongName from '../../../../utils/someCoolUtilWithARatherLongName';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import someCoolUtilWithARatherLongName
  from \\"../../../../utils/someCoolUtilWithARatherLongName\\";
"
`;

exports[`long-line.js 3`] = `
"import someCoolUtilWithARatherLongName from '../../../../utils/someCoolUtilWithARatherLongName';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import someCoolUtilWithARatherLongName
  from \\"../../../../utils/someCoolUtilWithARatherLongName\\";
"
`;

exports[`long-line.js 4`] = `
"import someCoolUtilWithARatherLongName from '../../../../utils/someCoolUtilWithARatherLongName';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import someCoolUtilWithARatherLongName
  from \\"../../../../utils/someCoolUtilWithARatherLongName\\";
"
`;

exports[`multiple_standalones.js 1`] = `
"import a, * as b from 'a';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import a, * as b from \\"a\\";
"
`;

exports[`multiple_standalones.js 2`] = `
"import a, * as b from 'a';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import a, * as b from \\"a\\";
"
`;

exports[`multiple_standalones.js 3`] = `
"import a, * as b from 'a';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import a, * as b from \\"a\\";
"
`;

exports[`multiple_standalones.js 4`] = `
"import a, * as b from 'a';
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import a, * as b from \\"a\\";
"
`;
